libusb/os/windows_winusb.c

```diff
@@ -2754,8 +2782,36 @@ static int winusbx_release_interface(int sub_api, struct libusb_device_handle *d
 	if (!HANDLE_VALID(winusb_handle))
 		return LIBUSB_ERROR_NOT_FOUND;
 
+	int initialized_iface;
+	if (priv->usb_interface[iface].is_associated_interface)	{
+		initialized_iface = priv->usb_interface[iface].first_associated_interface;
+	} else if (is_using_usbccgp){
+		initialized_iface = iface;
+	} else {
+		initialized_iface = 0;
+	}
+
+	usbi_dbg(ctx, "\tlibusb: release_interface(%p, %d) initialized_iface=%d claimed_count = %d\n", dev_handle, iface, initialized_iface, handle_priv->interface_handle[initialized_iface].claimed_interfaces_count);
```

Comment:
"\tlibusb:" seems odd.

Reply:
hah, some leftovers from debugging. Sorry for that